.github/workflows/genbuilds.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env xonsh
+"""Generates workflow files for each build matrix element. This is done
+so that we can restart indivual workflow elements without having to restart
+them all. Rerun this script to regenerate.
+"""
+from itertools import product
+import os
+import jinja2
+from pathlib import Path
+
+CURR_DIR = Path(__file__).parent
+environment = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(CURR_DIR),
+    trim_blocks=True,
+    variable_start_string="<%",
+    variable_end_string="%>",
```

Comment:
Why not use standard Jinja2 here?

Reply:
the usual ${{ }} brace is used by github actions

Reply:
Doesn't Jinja use just `{{}}`  I thought there was a way to escape this for literal `{{` and `}}`

Reply:
Yeah, I think either strings or raw can be used: https://stackoverflow.com/a/55464045

so `{% raw %}{{{% endraw %}` or `{{ "{{" }}`

Reply:
raw is better than quoting. wouldn't be confusing what is getting rendered by the script?

Reply:
I don't think so.  I think my issue is that I do so much jinja2 in my life that I am never going to remember alternative openers and closers 🤣 .

Reply:
Oh that :)

i was considerate about curly braces since there are more of Github than what is replaced by context.

Reply:
mako for jinja :)

Reply:
there are 13 `${{ ` . If you still prefer not to use custom tags,then I will change it.

Reply:
Yeah, I think for (my) sustainability I would prefer if this was in standard Jinja

Reply:
updated